controllers/controllers/resource/testdata/etcdadmcluster.yaml

```diff
@@ -1,28 +0,0 @@
-kind: EtcdadmCluster
```

Comment:
A lot of test files are getting deleted, is that intended?

Reply:
I removed these files with safe delete in GoLand. Unit tests are still passing. I also did a search for the filenames in our code base and nothing turned up. I thought it would be nice to clean up some unused test data files

Reply:
I see it was because we split it up between vsphere and cloudstack.